core/src/View/MDSpan/Kokkos_MDSpan_Header.hpp

```diff
@@ -54,6 +54,8 @@ struct index_pair_like<Kokkos::pair<IdxT1, IdxT2>, IndexType> {
 };
 }  // namespace detail
 }  // namespace Kokkos
+
+#define MDSPAN_IMPL_CHECK_PRECONDITION 0
 #include <mdspan/mdspan.hpp>
```

Comment:
Add a comment to capture your intent here, that this is the only location that includes header `<mdspan/mdspan.h>` and that the macro definition sets the desired configuration

Reply:
I talked to @crtrott  yesterday and I he does not want to do this (users can include mdspan themselves). We need to find some sort of alternative. Either way it will be complicated by the fact that users can include mdspan directly and have a different set of defines.